source/endpoints/transcript.rst

```diff
@@ -56,14 +56,25 @@ Code                   Description
                             }                           
                           ]
                         }       
+
+                        The json format of the transcript for a job with `skip_diarization` set to true will only show a single speaker.
 ---------------------- ---------------------------------------------------------------
  200                    ``Example Value``
 
                         **Accept: text/plain**
 
-                        `Speaker 1:    00:00    Hello there, this is a sample transcript in plain text`
+                        When the submitted job has `skip_diarization` either set to `false` or not set, the transcript will appear as follows:
+                        `Speaker 0    00:01 What is your name?`
```

Comment:
I wasn't able to run ReadTheDocs locally to verify the integrity of this restructured text format.